### ISSUE TITLE
Module not found: Error: Can't resolve 'rxjs/Subject' in 'node_modules/ngx-wow/services'

### ISSUE THREAD

**Issue by jacklaurencegaray** [tinesoft/ngx-wow#18]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### OS and Version?

macOS Mojave

### Versions

Angular CLI: 6.2.3
Node: 8.11.3
OS: darwin x64
Angular: 6.1.9
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.8.3
@angular-devkit/build-angular     0.8.3
@angular-devkit/build-optimizer   0.8.3
@angular-devkit/build-webpack     0.8.3
@angular-devkit/core              0.8.3
@angular-devkit/schematics        0.8.3
@angular/cli                      6.2.3
@ngtools/webpack                  6.2.3
@schematics/angular               0.8.3
@schematics/update                0.8.3
rxjs                              6.2.2
typescript                        2.9.2
webpack                           4.20.2

### Repro steps

N/A

### The log given by the failure

ERROR in ./node_modules/ngx-wow/services/wow.service.js
Module not found: Error: Can't resolve 'rxjs/Subject' in 'node_modules/ngx-wow/services'

### Desired functionality

In wow.service.d.ts,
Instead of `import { Subject } from 'rxjs/Subject'`, use `import { Subject } from 'rxjs'`.

In wow.service.js,
Instead of `import { Observable } from 'rxjs/Observable'`, use `import { Observable } from 'rxjs'`

The solutions above solve the problem. Ng serve does not run prior to the above steps.

**Comment by tinesoft** [comment]
Hi @jacklaurencegaray 

Thanks for opening the issue. This should be fixed quickly by migrating the library to `rxjs 6` too (i bet you are on Angular 6+, which relies on that version of `rxjs`).

I'll migrate the library to ng6 in  the coming days. In the mean time installing `rxjs-compat`, should fix that issue.

**Comment by tinesoft** [comment]
Closed by #21

**Comment by SamiHK** [comment]
Just replace import { Subject } from 'rxjs/Subject';
to : import { Subject } from 'rxjs';

Anywhere you see this issue on rxjs : 6.0.0 👍
